spec/outcome_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe Outcome do
+
+  it "should check for all types of horizontal wins" do
+    current_game = Outcome.new
+    top_horizontal_win = [["X", "O", "03"], ["04", "O", "O"], ["X", "X", "X"], ["X", "04", "X"], ["O", "O", "X"], ["03", "O", "X"], ["X", "O", "X"], ["03", "O", "X"]]
```

Comment:
Can you make these more readable by spacing your multi-dimensional array?

Reply:
done!